README.md

```diff
@@ -50,7 +50,7 @@ on the snap, including how to install it, please refer to [EdgeX snap](https://g
 
 #### Go
 
-- The current targeted version of the Go language runtime for release artifacts is v1.12.x
+- The current targeted version of the Go language runtime for release artifacts is v1.13.x
 - The minimum supported version of the Go language runtime is v1.11.x (currently v1.11.13)
```

Comment:
Minimum is also 1.13.

Reply:
Updated `README.md` for minimum version 1.13 ✅